cg/services/analysis_starter/configurator/file_creators/nextflow/params_file/raredisease.py

```diff
@@ -54,17 +60,19 @@ def _get_case_parameters(
     ) -> RarediseaseParameters:
         """Return case-specific parameters for the analysis."""
         analysis_type: str = self._get_data_analysis_type(case_id)
-        target_bed_file: str = self._get_target_bed(case_id=case_id, analysis_type=analysis_type)
-        skip_germlinecnvcaller: bool = self._get_germlinecnvcaller_flag(analysis_type=analysis_type)
+        target_bed_file: str = self._get_target_bed_from_lims(case_id)
+        sample_mapping_path: Path = self.extension.sample_mapping_file_creator.get_file_path(
```

Comment:
this is why we need the extension into the param creator: The Parameters must have the sample mapping path